example/test@test.priv

```diff
@@ -1 +1 @@
-684e2aaadf3149610d0d1ebb827a565d0faf2078b821b6042d6cedffe18d0d7424780113f7dc5bbcad0a0e98da919e1c7b795346d4f09f15091b8117cec9b57d
\ No newline at end of file
+4209ba343a92f4d086921a3f3c1eb26f50f2fece610ec3524058de79281564c2
```

Comment:
again, autogenerated file should not be in git

Reply:
It was decided long time before, that we will keep those files in test

Reply:
We have to revise this decision

Reply:
We should improve CLI and get rid of these files. Until the moment, we should keep them for fast deployment for testing purposes @Warchant